giraffe/src/utils/getBandHoverIndices.ts

```diff
@@ -91,32 +40,58 @@ export const getBandHoverIndices = (
   lineLengths: LineLengths,
   hoverRowIndices: number[],
   hoverGroupData: NumericColumnData,
-  bandBoundaries: BandBoundaries
+  bandLineIndexMap: object
 ): BandIndexMap => {
   const bandHoverIndices = {
     rowIndices: [],
     lowerIndices: [],
     upperIndices: [],
   }
+  const hoverGroupBoundaries = {}
+
+  if (Array.isArray(hoverGroupData)) {
+    hoverGroupData.forEach((lineIndex, position) => {
```

Comment:
i think i'd like to start standardizing counters to be `i` since it's kind of an industry standard. what do you think about changing `position` to `i`?

Reply:
Done.